kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -271,21 +249,68 @@ define([
             this.$elem.append(errorElem);
         },
 
+        /**
+         * Identifies any potential file type mappings based on a call to the Staging Service,
+         * and returns the list of staging file objects with an added 'mappings' field.
+         * @param {Array} stagingFiles
+         * @returns {Array} the stagingFiles array augmented with a 'mappings' field for each file,
+         *  each containing an id, title, and app_weight. If there are no file->type mappings, this
+         *  field is null
+         */
         identifyImporterMappings: function (stagingFiles) {
-            /*
-                Add a list of top matches for each file, sorted by weight
-            */
             const fileNames = stagingFiles.map((f) => f['path']);
-            const fileList = $.param({ file_list: fileNames }, true);
-            let mappings = [];
-            return Promise.resolve(
-                this.stagingServiceClient.importer_mappings({
-                    file_list: fileList,
-                })
-            )
+
+            /* Currently, the mapping info lookup is only a GET request with URL parameters.
```

Comment:
😬 
I hope the staging area gets changed soon...

Reply:
erk!!